build.gradle

```diff
@@ -184,7 +184,8 @@ sonarqube {
         property "sonar.pitest.reportsDirectory", "build/reports/pitest"
         property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/**," +
                                      "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/SystemDateProvider.java," +
-                                     "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/Application.java"
+                                     "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/Application.java," +
+                                     "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/**"
```

Comment:
I know this is boilerplate and I completely agree with the exclusion but we might need to get @raghera 's approval too.